notebooks/metatranscriptomics_pipeline_iSeqnorm.ipynb

```diff
@@ -0,0 +1,1477 @@
+    "                    color_map='viridis',\n",
+    "                    annot_fmt='')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 4: calculate pooling values for MiniPico with autopool\n",
+    "\n",
+    "This step will calculate the sample pooling, and update the sample data frame with the pool info.\n",
+    "There are two automated methods to pool:\n",
+    "1. **norm**: This will attempt to generate a normalized pool, automatically infering the best parameter for pooling.\n",
+    "    - ***pool_failures***:\n",
+    "        - _high_: will pool failures at the highest pooling volume from optimized pooling.\n",
+    "        - _low_: will pool failures at the lowest pooling volume from optimized pooling.\n",
+    "\n",
+    "2. **evp**: This will pool an even volume per sample.\n",
+    "    - ***total_vol***: (Optional, Default: 100µL) The total volume to pool, in uL. Each sample will be pooled at 1/N of that volume.\n",
+    "\n",
+    "3. **automate**: (Optional, Default = True) When False, this argument will allow one input parameters for **Legacy** arguments. \n",
+    "\n",
+    "> **Legacy**\n",
+    "> There are legacy parameters to control pooling behaviors when autopool automation (automate=True) returns a poor result. To use these parameters, one must pass automate=False.\n",
+    "\n",
+    ">   - **min_conc**: (default: 0) This is the minimum concentration for a sample to be considered for pooling.\n",
+    "    Set to 0 to pool all samples, regardless of concentration. Increasing this will have the \n",
+    "    effect of increasing pool concentration, at the expense of samples dropping out. \n",
+    ">   - **floor_conc**: This is the lowest concentration equivalent for which a sample will be \n",
+    "    accurately pooled. Samples below this concentration will be pooled to the volume that they \n",
+    "    would have been if they were actually that concentration. For example, if `floor_conc=20`, \n",
+    "    and a sample at 20 nM pools at 500 nL, a sample at 40 nM will pool at 250 nL but a sample at \n",
+    "    10 nM will still pool at 500 nL (rather than 1000). Increasing this value will have the effect \n",
+    "    of increasing pool concentration, but decreasing read counts for low-concentration samples. \n",
+    ">   - **total_nmol**: This is the total number of molecules to shoot for in the pool. Increasing\n",
+    "    this will increase the overall volume of the pool.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Calculate and plot pooling volumes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "threshold = find_threshold(plate_df['MiniPico Library Concentration'], plate_df['Blank'])\n",
+    "plate_df = autopool(plate_df,method='evp',total_vol=190)\n",
```

Comment:
When I wrote the function 100 µL seemed like a good default value. After putting this function in practice, we figured that pooling 190µL in total would result in ~500nL per sample for a full 384 well plate, so we hardcoded the value in the notebook to 190µL, since that is the value that wetlab technicians were using most often. I can change the default value to 190µL.